build/config/compiler/BUILD.gn

```diff
@@ -283,9 +288,12 @@ config("compiler") {
 
     cflags += common_mac_flags
 
-    # Without this, the constructors and destructors of a C++ object inside
-    # an Objective C struct won't be called, which is very bad.
-    cflags_objcc = [ "-fobjc-call-cxx-cdtors" ]
+    cflags_objcc += [
+      "-std=c++11",
```

Comment:
kind of a bummer that this isn't next to the place that sets c++11 for cflags_cc, since this'll be hard to update when we want to change that (say to c++14 or whatever).